python/cudf/cudf/_lib/lists.pyx

```diff
@@ -162,6 +162,26 @@ def contains_scalar(Column col, object py_search_key):
     return result
 
 
+def index_of_scalar(Column col, object py_search_key):
```

Comment:
Minor nit: I assume this naming is to match `contains_scalar`? IMO the `_scalar` suffix is redundant.